### ISSUE TITLE
[application-header-layout] Missing fi-FI Translations

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-framework#311]
# Bug Report

## Description
Missing fi-FI translations

**Comment by yuderekyu** [comment]
those are just placeholder translations, not actual ones right?

**Comment by emilyrohrbough** [comment]
Yes, they are 'mock' translations but are throwing errors for consuming projects since it is one of the terra-supported locales
